packages/editor-ui/src/hooks/posthog.ts

```diff
@@ -0,0 +1,426 @@
+// version 0.1.0
+
+import { usePostHog } from '@/stores/posthog';
+import {
+	hoosGetPosthogAppendNoCaptureClasses,
+	hooksPosthogSetMetadata,
+} from '@/hooks/posthog/index';
+import { ComponentPublicInstance } from 'vue';
+import { ITelemetryTrackProperties } from 'n8n-workflow';
+import { IPersonalizationLatestVersion } from '@/Interface';
+
+window.featureFlag = {
```

Comment:
is this getting used?

Reply:
would remove this.. it's values are wrong anyways.. we don't depend on this since we moved feature flag evaluation serverside

Reply:
reimplemented here just for testing (with a different interface) https://github.com/n8n-io/n8n/pull/5642/files#diff-71a4eee50c3824c3d17a6a53f7497ebf292c3a10e712df3fdbc1a8f971e72e3cR53

Reply:
Nice! I've removed it from here since it's the wrong value anyway.